lib/src/value/calculation.dart

```diff
@@ -162,17 +169,23 @@ class SassCalculation extends Value {
   /// a [CalculationInterpolation].
   static Object operate(
           CalculationOperator operator, Object left, Object right) =>
-      operateInternal(operator, left, right, inMinMax: false);
+      operateInternal(operator, left, right,
+          inMinMax: false, inSupportsDeclaration: false);
 
   /// Like [operate], but with the internal-only [inMinMax] parameter.
   ///
   /// If [inMinMax] is `true`, this allows unitless numbers to be added and
   /// subtracted with numbers with units, for backwards-compatibility with the
   /// old global `min()` and `max()` functions.
+  ///
+  /// If [inSupportsDeclaration] is `true`, no simplification will be done.
```

Comment:
I'd call this parameter `simplify`, since that more directly represents what it *does* rather than what the context is.